src/TxTemplate.hs

```diff
@@ -114,7 +115,7 @@ enforceEqualArrayLens = go [] Nothing False . filter isArrWithMultiple
               newMismatch = isMismatch || maybe False (/= len) mFirstLen
           in go (msg : lenStrs) (mFirstLen <|> Just len) newMismatch ps
         _ -> go lenStrs mFirstLen isMismatch ps
-    isArrWithMultiple (_,MU.Array a) = V.length a >= 1
```

Comment:
If this is asking whether it's an array with multiple member, is it still correct to say only `>= 1`?

Reply:
`> 1` is the desired behavior here.  This is now covered by a new unit test and golden test:

https://github.com/kadena-io/kda-tool/pull/18/files#diff-cf1488059321e429d597cd621a2efcf1ada540ec58f06f5af0fffa05aeb7c75eR32
https://github.com/kadena-io/kda-tool/pull/18/files#diff-cf1488059321e429d597cd621a2efcf1ada540ec58f06f5af0fffa05aeb7c75eR35